web/app/scripts/utils/date-utils.js

```diff
@@ -0,0 +1,154 @@
+(function() {
```

Comment:
Now that I've done some work on date localization myself, I think that this file would be more appropriate in `scripts/localization` where there is already some Date-related localization code.